### ISSUE TITLE
ERROR: 'Invalid Token'

### ISSUE THREAD

**Issue by JoseAlberto218** [FlashChaser/Open-Source-Battle-Bot#4]
So I tried to install this bot and when I get to the part where it's supposed to create a new account, just after finishing part 6 of the tutorial, it exists with an error code that says: {ERROR: 'Invalid Token'} and something about 'request is expired'. I tried to fix it but it kept happening and I've been seeing the code and I could not find when this triggers

**Comment by pawelorzech** [comment]
So you've compiled it by yourself and then it fails to complete tutorial? You're trying to create account on JP or GLB? Have you tried again, later?

**Comment by JoseAlberto218** [comment]
Not exactly, I downloaded the bot and tried to transfer an account. Then it gave me an error code and exit the console. And it does that every time. I think the problem is with the request from Bandai's server but I don't know why it is happening.

And I tried to create both, JP and GLB, none of them worked. And yeah, I also tried later but nothing happened. So far, I am the only one with this problem from what I have read.
